API.md

```diff
@@ -583,9 +583,9 @@ As stated at the beginning of the document, `--ignore` parameter is an alias for
 
 **lab** uses a shareable [eslint](http://eslint.org/) config, and a plugin containing several **hapi** specific linting rules. If you want to extend the default linter you must:
```

Comment:
Shouldn't we remove the mention of the shareable config since it is not directly exposed now?

Reply:
I updated the wording so it only reference the plugin (with a config).